src/components/Outline.js

```diff
@@ -36,8 +37,21 @@ class Outline extends Component {
     }
   }
 
+  selectItem(location) {
+    const selectedSourceId = this.props.selectedSource.get("id");
```

Comment:
generally we destructure our props so that it is clear what our `this` deps are in the func

```
const {selectedSource, selectSource} = this.props;
```